### ISSUE TITLE
Missed styles for tfoot

### ISSUE THREAD

**Issue by barmaley443** [daniel-nagy/md-data-table#31]
it will be nice to style tfoot element used for summary info

**Comment by barmaley443** [comment]
https://github.com/daniel-nagy/md-data-table/pull/32

**Comment by barmaley443** [comment]
example
```
        <md-data-table-container>
            <table md-data-table>
                <thead md-order="order" md-trim-column-names>
                    <tr>
                        <th order-by="retailerName">Name</th>
                        <th numeric order-by="count">Count</th>
                        <th numeric unit="usd" precision="2" order-by="cost">Sum</th>
                    </tr>
                </thead>
                <tbody>
                    <tr ng-repeat="stat in statistics | orderBy: order" ng-init="getTotal(statistics)">
                        <td>{{stat.retailerName}}</td>
                        <td>{{stat.count}}</td>
                        <td>{{stat.cost}}</td>
                    </tr>
                </tbody>
                <tfoot>
                    <tr>
                        <td><strong>Total:</strong></td>
                        <td class="numeric">{{total.count}}</td>
                        <td class="numeric">{{total.cost}}</td>
                    </tr>
                </tfoot>
            </table>
        </md-data-table-container>
```

**Comment by daniel-nagy** [comment]
Resolved in version 0.5.0
